Expose `packagePath` as a getter in FlutterCommand.

chingjun opened the pull request:
So that it can be overridden internally.

chingjun committed:
Expose `packageFilePath` as a getter in FlutterCommand.

So that it can be overridden internally.

jonahwilliams reviewed (commented):
Can you leave a doc comment on packagesFilePath? At least so that we know it is overriden in case we forget in 1+ years.

I would also like a simple test case that shows this still works. You assert that an overriden and fixed packages file path is used for the build info instead of the field passed via the command line?

chingjun committed:
Add test and comment.

Also renamed packagesFilePath to packagesPath since it is more commonly
used in the code base.

chingjun commented:
Added test and comment. Also renamed `packagesFilePath` to `packagesPath` since it is more commonly used in the code base.

PTAL, thanks!

jonahwilliams reviewed (approved):
LGTM thank you!
